docs/learn/learn-staking-miner.md

```diff
@@ -0,0 +1,113 @@
+the sequential Phragmén algortihm to produce results, and the result is then sent as a transaction
+to the relay chain via a normal signed extrinsic. The transaction requires a bond, and a transaction
+fee. The best solution is rewarded which in the least covers the transaction fee, and the bond is
+returned to the account. If the solution is not valid, the bond and the fee are lost.
+
+Staking miner uses a pallet called `pallet_election_provider_multi_phase` and can only produce
+solutions during the signed phase of the pallet's life cycle. Once the signed phase is over and the
+unsigned phase starts, only the off-chain workers can provide election results.
+
+## NPoS election optimization
+
+A basic election solution is a simple distribution of stake across validators, but this can be
+optimized for better distribution equaling a higher security score. The staking miner does not act
+as a validator and focuses solely on the election result and optimization of the solution. It
+connects to a specified chain and keeps listening to new signed phase of the election pallet in
+order to submit solutions to the NPoS election. When the correct time comes, it computes its
+solution and submits it to the chain. The default miner algorithm is sequential Phragmén with a
+configurable number of balancing iterations that improve the score.
+
+Running the staking miner requires passing the seed of a funded account in order to pay the fees for
+the transactions that will be sent. The same account's balance is used to reserve deposits as well.
+The best solution in each round is rewarded. All correct solutions will get their bond back and the
+ones that submit invalid solutions will lose their bond.
+
+![NPoS election optimization](../assets/staking-miner/NPoS-election-optimization.png)
+
+## Signed Phase of the election pallet
+
+The election provider pallet `pallet_election_provider_multi_phase` is divided into two phases,
+**signed** and **unsigned**. At the end of the pallet's timeline, the function `elect()` is called.
+
+```
+                                                                   elect()
+                +   <--T::SignedPhase-->  +  <--T::UnsignedPhase-->   +
+  +-------------------------------------------------------------------+
+   Phase::Off   +       Phase::Signed     +      Phase::Unsigned      +
+```
+
+Solutions provided by the staking miner can only be submitted during the signed phase. Solutions are
+submitted and queued on the chain as a `RawSolution`. Once submitted, a solution cannot be retracted
+by the originating account.
+
+`RawSolution` struct definition:
+
+```
+pub struct RawSolution<S> {
+    pub solution: S, // The solution itself
+    pub score: ElectionScore, // The claimed score of the solution.
+    pub round: u32, // The round at which this solution should be submitted.
+}
+```
+
+A maximum of `pallet::Config::MaxSignedSubmissions` will be stored on-chain and they will be sorted
```

Comment:
You should mention how much this value is in Polkadot and Kusama, and link to it.